lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if Gem::Version.new(GettextSetup::VERSION) < Gem::Version.new('0.8')
```

Comment:
@Magisus — After reading the reason for this being added in the first place, can this conditional be removed altogether? The Gemfile specifies version >= 0.10. Is there still a situation where an earlier version of gettext-setup would run this code?